adal/src/main/java/com/microsoft/aad/adal/BrokerProxy.java

```diff
@@ -110,9 +110,12 @@
      * Verifies the broker related app and AD-Authenticator in Account Manager
      * ADAL directs call to AccountManager if component is valid and present. It
      * does not direct call if the caller is from Authenticator itself.
```

Comment:
This documentation is now incorrect, right? Can you now clarify the expected functionality here?